### PR TITLE
Add s3 tests

### PR DESCRIPTION
**Pull request by Alipayy**
Signed-off-by: a.lipay <a.lipay@yadro.com>

### CODE REVIEW

File: pytest_tests/steps/load.py [hunk 1]
```diff
@@ -16,6 +18,55 @@ def get_services_endpoints(
...
 
+@allure.title("Init s3 client")
+def init_s3_client(load_nodes: list, login: str, pkey: str, hosting: Hosting):
+    public_keys = ""
+    service_configs = hosting.find_service_configs(STORAGE_NODE_SERVICE_NAME_REGEX)
+    for service_config in service_configs:
+        host = hosting.get_host_by_service(service_config.name)
+        public_key = (
+            host.get_shell()
+            .exec("sudo neo-go wallet dump-keys -w /etc/neofs/s3/wallet.json | head -2 | tail -1")
```
[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py:29]: will do

[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py:29]: 1 - Checked. We already have this in common.py (NEOGO_EXECUTABLE = os.getenv("NEOGO_EXECUTABLE", "neo-go"),
NEOFS_AUTHMATE_EXEC = os.getenv("NEOFS_AUTHMATE_EXEC", "neofs-authmate")) So, will use it.

[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py]: done

File: pytest_tests/steps/load.py [hunk 2]
```diff
@@ -16,6 +18,55 @@ def get_services_endpoints(
...
+        public_keys += f"--gate-public-key {public_key} "
+    node_endpoint = service_configs[0].attributes["rpc_endpoint"]
+    password_input = [InteractiveInput(prompt_pattern="Enter password for.*", input="")]
+    for load_node in load_nodes:
+        ssh_client = SSHShell(host=load_node, login=login, password=pkey)
+        path = ssh_client.exec(r"sudo find . -name 'k6' -exec dirname {} \; -quit").stdout.strip(
+            "\n"
+        )
+        cmd = (
+            f"neofs-s3-authmate issue-secret --wallet {path}/scenarios/files/wallet.json "
```
[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py]: done

File: pytest_tests/steps/load.py [hunk 3]
```diff
@@ -16,6 +18,55 @@ def get_services_endpoints(
...
+                "aws_access_key_id"
+            )
+        )
+        aws_secret_access_key = str(
+            re.search(r"secret_access_key.*:\s.(?P<aws_secret_access_key>\w*)", output).group(
+                "aws_secret_access_key"
+            )
+        )
+        configure_input = [
+            InteractiveInput(prompt_pattern=r"AWS Access Key ID.*", input=aws_access_key_id),
```
[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py:64]: done

File: pytest_tests/steps/load.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py]: ohh..ok :) will fix

**Review by jingerbread — APPROVED** [review]

File: pytest_tests/steps/load.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py]: done, but need to fix neofs-testlib before merge...

[the remark this answers is not in the document]
Comment (Alipayy) [reply pytest_tests/steps/load.py]: we can merge, works with neofs-testlib 0.5.0

**Pull request merged**